src/components/MovieTrailer/MovieTrailer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import './MovieTrailer.css'
+
+const MovieTrailer = ({ movieKey }) => (
+  <div className="video-responsive">
+    <iframe
+      // width="853"
+      // height="480"
+      src={`https://www.youtube.com/embed/${movieKey}`}
+      frameBorder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+      allowFullScreen
+      title="Embedded youtube"
+    />
+  </div>
+);
+
+MovieTrailer.propTypes = {
+  movieKey: PropTypes.string.isRequired
```

Comment:
Great work using `PropTypes` here! I like that you've added the `isRequired` property too!